src/index.js

```diff
@@ -2,6 +2,13 @@
  * A single entry-point for all of the external-facing functionality.
  */
 
+import "../less/echo.less"
+import "../less/main.less"
+import "../less/overrides.less"
+import "../less/popover.less"
+
+import "mathquill/build/mathquill.css";
```

Comment:
We need to include all of the .less/.css files here so that webpack knows about them and can bundle them into math-input.css.